sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -121,9 +121,9 @@ def __repr__(self) -> str:
                 task.cancel()
         if to_cancel_partitions:
             _LOGGER.info(
-                "EventProcesor %r has cancelled partitions %r",
+                "EventProcessor %r has cancelled partitions %r",
```

Comment:
This becomes "to cancel" instead of "has cancelled"

Reply:
new log added